.travis.yml

```diff
@@ -7,4 +7,4 @@ install:
 before_script:
   - flake8 .
 script:
-  - python ./init.py
+  - python test_googleMapsApiModule.py

```

Comment:
Can we have a regex or something similar in file name to run all files starting with `test_`

Reply:
@Mansi145 - If I'm not wrong, pytest can be triggered through cli itself like `pytest` and all `test_*.py` files get automatically executed. Check it out once

Reply:
Okay @athityakumar, I'll check it out.

Reply:
@vishakha-lall @athityakumar Fixed it. Please review.